build.sh

```diff
@@ -114,7 +114,7 @@ function buildAll {
 }
 
 function buildLibCudfJniInDocker {
-    local cudaVersion="11.8.0"
+    local cudaVersion="12.9.0"
```

Comment:
Can we use `12.9.1`? In `spark-rapids-jni`, we already switched to it.

Reply:
Changed it to 12.9.1